base/pmap.jl

```diff
@@ -64,9 +58,13 @@ Split a collection into at least `min_batch_count` batches.
 Equivalent to `partition(c, max_batch_size)` when `length(c) >> max_batch_size`.
 """
 function batchsplit(c; min_batch_count=1, max_batch_size=100)
+    if min_batch_count < 1
+        throw(ArgumentError("min_batch_count must be > 0, got $min_batch_count"))
+    end
 
-    @assert min_batch_count > 0
-    @assert max_batch_size > 1
+    if max_batch_size < 1
```

Comment:
"Probably a bug" is not an error. It's convenient to compare the degenerate case through the same code path with a parameter instead of having to change syntax. Real bugs can be identified via static analysis and testing.